### PR TITLE
Fix typos and rephrase staging workflow documentation

### PR DESCRIPTION
**Pull request by saraycp**

### CODE REVIEW

File: xml/obs_best_practice_staging_workflow.xml [hunk 1]
```diff
@@ -20,10 +20,10 @@
   <para>
     The maintenance of those packages can be made on a collaborative way via
     Build Service. As shown in the following diagram, the maintainer can create
-    a package and then she or any other developer can branch it (make a copy of
-    it), do some changes on its code and request those changes to be applied on
-    the original package. After that, the maintainer usually reviews the
-    request, chat with the developer in case it needs some fixes and end up
+    a package and then he or any other developer can branch it (make a copy of
```
Comment (sknorr) [inline xml/obs_best_practice_staging_workflow.xml:23]: There was a phase in the 2000's where people used either "he or she" or just "she" to be gender-inclusive.
These days, the more common gender-inclusive form in English is just using "they".

File: xml/obs_best_practice_staging_workflow.xml [hunk 2]
```diff
@@ -258,7 +258,7 @@
      </mediaobject>
     </figure>
     <para>
-      From the configuration page it is possible to delete a Staging Project,
+      From the configuration's page it is possible to delete a Staging Project,
```
Comment (sknorr) [inline xml/obs_best_practice_staging_workflow.xml:261]: This correction seems wrong -- I'd suggest to just revert this line.

**Comment by saraycp** [comment]
@sknorr thanks for your suggestions. I've  addressed what you propose. Could you have another look?

File: xml/obs_best_practice_staging_workflow.xml [hunk 3]
```diff
@@ -48,29 +48,29 @@
...
     conflicts with other packages. To deal with these situations, Build Service
     provides what we call Staging Workflow.
   </para>
   <para>
     The idea behind the Staging Workflow is testing the requests incrementally
     by batches. First, a copy of the original project is created, it is called
     Staging Project and is going to act as a playground. The Staging Manager
-    selects some of the requests he consider they belong together and assign the
-    corresponding packages to the Staging Project. This way, the group of
+    selects some of the requests they considers they belong together and assigns the
```
Comment (sknorr) [inline xml/obs_best_practice_staging_workflow.xml:59]: Grammar fix: `they` needs to go with `consider`; wording improvement for `belong`:
```suggestion
    selects some of the requests they consider to be belonging together and assigns the
```

File: xml/obs_best_practice_staging_workflow.xml [hunk 4]
```diff
@@ -48,29 +48,29 @@
...
+    corresponding packages to the Staging Project. This way, the groups of
     packages are going to be tested (built) in one go. Once the Staging Project
     gets built, the changes can be merged to the original project.
   </para>
   <para>
-    The Staging Manager can create as many Staging Projects as he consider and
-    assign different selections of requests to each of them. It is still tedious
+    The Staging Manager can create as many Staging Projects as they considers and
```
Comment (sknorr) [inline xml/obs_best_practice_staging_workflow.xml:65]: Grammar fix (they + 3rd person plural verb) and use a better verb than consider:
```suggestion
    The Staging Manager can create as many Staging Projects as they require and
```

**Comment by saraycp** [comment]
> The one slightly unintuitive twist in the use of "they" is that it _still_ requires a 3rd-person plural verb to go with it, even though we are talking about a single person. :)

I read about this yesterday because I always doubt what is the best approach. It seems there is a tendency to use "they" instead of "he/she" but keeping the verb like it was (3rd person singular). [This article](https://jakubmarian.com/he-or-she-vs-they-in-english/) gives some good examples.

However, it's true it sounds weird, so I can try to rephrase everything using 3rd person plural verbs.

**Comment by saraycp** [comment]
@sknorr I've just rephrased the text to always use 3rd person plural.

**Pull request merged**